test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/project.json

```diff
@@ -26,10 +26,12 @@
         "dotnet-test-xunit": "1.0.0-dev-*"
       }
     },
-    "dnx451": {
+    "net451": {
+      "frameworkAssemblies": {
+        "System.Threading.Tasks": ""
```

Comment:
Does this need `{ "type": "build" }` like in https://github.com/aspnet/HttpAbstractions/pull/579?